README.md

```diff
@@ -204,8 +206,10 @@ You may encounter an issue with the `unused_deps` check failing on the CI while
 
 ## Changelog
 
-See [CHANGELOG.md](https://github.com/karolsluszniak/ex_check/blob/master/CHANGELOG.md).
+See [CHANGELOG.md](./CHANGELOG.md).
 
-## License
+## Copyright and License
 
-See [LICENSE.md](https://github.com/karolsluszniak/ex_check/blob/master/LICENSE.md).
+Copyright (c) 2019 Karol Słuszniak
+
+This library is released under the MIT License. See the [LICENSE.md](./LICENSE.md) file.
```

Comment:
Same here - README links should work on HexDocs too.

Reply:
Likewise